src/main/fc/runtime_config.c

```diff
@@ -59,6 +59,7 @@ const char *armingDisableFlagNames[]= {
     "DSHOT_BBANG",
     "NO_ACC_CAL",
     "MOTOR_PROTO",
+    "CRASHFLIP",
```

Comment:
This is fragile and will break again.

Either:
```suggestion
    [LOG2(ARMING_DISABLED_CRASHFLIP)] = "CRASHFLIP",
```

Or, possibly better:
``` c
// in runtime_config.h
extern const char *armingDisableFlagNames[];
// just after armingDisableFlagNames
STATIC_ASSERT(ARRAYLEN(armingDisableFlagNames) == ARMING_DISABLE_FLAGS_COUNT, "armingDisableFlagNames size mismatch");
```

And `armingDisableFlagNames` should probably be hidden in getter function (`const char* armingDisableFlagName( armingDisable flag)`, returning name based on (probably) lowest bit set (`flag & -flag`)